### PR TITLE
[GEOS-7309]: Setting explicit joda-time version

### PR DESCRIPTION
**Pull request by dromagnoli**

### CODE REVIEW

File: src/pom.xml [hunk 1]
```diff
@@ -1617,6 +1622,7 @@
   <gs.version>2.9-SNAPSHOT</gs.version>
   <gt.version>15-SNAPSHOT</gt.version>
   <gwc.version>1.9-SNAPSHOT</gwc.version>
+  <jodatime.version>2.2</jodatime.version>
```
Comment (jodygarnett) [inline src/pom.xml]: This is just a single jar right? could leave it in the dependency management section ...

**Comment by aaime** [comment]
Daniele, looks good, but don't we have a Jira ticket for this one? (the commit message does not refer one?)

**Comment by dromagnoli** [comment]
Mmm. Nope. I think that I just prepared the pull-request after the mail
discussion.
I can create it now and update the message.

On Sat, Nov 7, 2015 at 5:29 PM, Andrea Aime <notifications@github.com>
wrote:

> Daniele, looks good, but don't we have a Jira ticket for this one? (the
> commit message does not refer one?)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geoserver/geoserver/pull/1315#issuecomment-154721437>.
>



-- 
==
GeoServer Professional Services from the experts! Visit
http://goo.gl/it488V for more information.
==

Ing. Daniele Romagnoli
Senior Software Engineer

GeoSolutions S.A.S.
Via Poggio alle Viti 1187
55054  Massarosa (LU)
Italy
phone: +39 0584 962313
fax:      +39 0584 1660272

http://www.geo-solutions.it
http://twitter.com/geosolutions_it

-------------------------------------------------------

*AVVERTENZE AI SENSI DEL D.Lgs. 196/2003*

Le informazioni contenute in questo messaggio di posta elettronica e/o
nel/i file/s allegato/i sono da considerarsi strettamente riservate. Il
loro utilizzo è consentito esclusivamente al destinatario del messaggio,
per le finalità indicate nel messaggio stesso. Qualora riceviate questo
messaggio senza esserne il destinatario, Vi preghiamo cortesemente di
darcene notizia via e-mail e di procedere alla distruzione del messaggio
stesso, cancellandolo dal Vostro sistema. Conservare il messaggio stesso,
divulgarlo anche in parte, distribuirlo ad altri soggetti, copiarlo, od
utilizzarlo per finalità diverse, costituisce comportamento contrario ai
principi dettati dal D.Lgs. 196/2003.



The information in this message and/or attachments, is intended solely for
the attention and use of the named addressee(s) and may be confidential or
proprietary in nature or covered by the provisions of privacy act
(Legislative Decree June, 30 2003, no.196 - Italy's New Data Protection
Code).Any use not in accord with its purpose, any disclosure, reproduction,
copying, distribution, or either dissemination, either whole or partial, is
strictly forbidden except previous formal approval of the named
addressee(s). If you are not the intended recipient, please contact
immediately the sender by telephone, fax or e-mail and delete the
information in this message that has been received in error. The sender
does not give any warranty or accept liability as the content, accuracy or
completeness of sent messages and accepts no responsibility  for changes
made after they were sent or for other risks which arise as a result of
e-mail transmission, viruses, etc.

**Comment by dromagnoli** [comment]
Done

**Pull request merged**